src/main/java/org/dcsa/ovs/notifications/model/MailTemplate.java

```diff
@@ -0,0 +1,38 @@
+package org.dcsa.ovs.notifications.model;
+
+import lombok.Data;
+import org.dcsa.core.events.model.Event;
+import org.dcsa.core.events.model.enums.EventClassifierCode;
+import org.dcsa.core.events.model.enums.EventType;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import java.util.Set;
+
+@Data
+public class MailTemplate {
+
+    @Email
+    private String to = "NOT_SPECIFIED";
+
+    @NotBlank
+    private String subject;
+
+    @NotBlank
+    private String body;
+
+    private Set<EventClassifierCode> onlyForEventClassifierCode;
+
+    private Set<EventType> onlyForEventType;

```

Comment:
Wow ;-)

Reply:
Wow? :)

Reply:
I just thought it was cool that we support only sending mails for some event types :-)